### PR TITLE
#167164985 View User Profile

### PR DESCRIPTION
**Pull request by allebd**
#### Pivotal tracker story
[#167164985](https://www.pivotaltracker.com/story/show/167164985)
#### What does this PR do?
View user profile
#### Summary of Task
- write using test to view user profile by userId
- create a route to view profile
- create user controller to view profile
#### How can this be tested?
- Use **git clone** to clone this repository to local branch
- Use **npm install** to download all necessary packages
- Ensure your database is created or you can use **sequelize db:create** if database does not exist
- Use **npm run test**  to test
- Go your POSTMAN and type **http://localhost:${PORT}/api/v1/profiles/:userId** to view a particular user by userId
#### Screenshots (if appropriate)
Screenshot 2019-08-02 at 3 49 02 PM
Screenshot of a successful view of user profile

Screenshot 2019-08-02 at 3 55 27 PM
Screenshot of a user profile not found

### CODE REVIEW

**Review by OvieMudi — APPROVED** [review]

**Review by OvieMudi — APPROVED** [review]
LGTM

File: src/routes/user.js [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+const { signUp, profile } = userController;
 const user = express.Router();
 
-const middlewares = [userValidator.signUp];
+// Route to sign up a user
+user.post('/users', signUpValidator, signUp);
 
-user.post('/users', middlewares, signUp);
+// Route to get user profiles by username
+user.get('/profiles/:username', profile);
```
Comment (Ukhu) [inline src/routes/user.js]: I noticed there is no validation middleware on this route to ensure that the username entered isn't empty and is valid.

Comment (allebd) [reply src/routes/user.js]: Noted, would work on it

Comment (allebd) [reply src/routes/user.js]: Done

**Review by benisi — APPROVED** [review]
LGTM

**Review by Nerocodes — APPROVED** [review]
LGTM

**Review by OvieMudi — APPROVED** [review]
LGTM

**Pull request merged**